tests/slack/test_slack_source.py

```diff
@@ -154,3 +191,32 @@ def test_users(destination_name: str) -> None:
     print(table_counts.keys())
     assert set(table_counts.keys()) >= set(expected_tables)
     assert table_counts["users"] >= 300  # The number of users can increase over time
+
+
+@pytest.mark.parametrize("destination_name", ALL_DESTINATIONS)
+def test_private_channels(destination_name: str) -> None:
+    pipeline = dlt.pipeline(
+        pipeline_name="slack",
+        destination=destination_name,
+        dataset_name="slack_data",
+        dev_mode=True,
+    )
+    PRIVATE_CHANNEL_NAME = "test-private-channel"
+    source = slack_source(
+        start_date=pendulum.now().subtract(weeks=1),
+        end_date=pendulum.now(),
+        selected_channels=[PRIVATE_CHANNEL_NAME],
+        include_private_channels=True,
+        replies=True,
+    ).with_resources(PRIVATE_CHANNEL_NAME, f"{PRIVATE_CHANNEL_NAME}_replies")
+    load_info = pipeline.run(source)
+    assert_load_info(load_info)
+    table_names = [t["name"] for t in pipeline.default_schema.data_tables()]
+
+    expected_message_table_name = f"{PRIVATE_CHANNEL_NAME}_message".replace("-", "_")
+    expected_replies_table_name = f"{PRIVATE_CHANNEL_NAME}_replies_message".replace(
+        "-", "_"
+    )
+
+    assert expected_message_table_name in table_names
+    assert expected_replies_table_name in table_names
```

Comment:
Will this table be created if we have zero messages available?

Reply:
ah, because in three months the old messages will go away..good point. i guess we don't really need the replies test here